runtime/development/Cargo.toml

```diff
@@ -287,6 +292,7 @@ try-runtime = [
     "pallet-fees/try-runtime",
     "pallet-identity/try-runtime",
     "pallet-interest-accrual/try-runtime",
+    "pallet-liquidity-rewards/runtime-benchmarks",
```

Comment:
We must be a bit more cautious with feature flags @lemunozm and @mikiquantum. I guess development would not build with try-runtime in this scenario separately. Just am marking this here, as these feature flags cost us quite some time, then they are set wrongly and some way down the line in the future make problems. 

This also goes for myself. Maybe we should brainstorm, how we could enforce correct feature flags and what should be the default setup for pallets and their features.